src/test/java/Homework20.java

```diff
@@ -0,0 +1,30 @@
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+//Removed all thread.sleep from BaseTest and previous homeworks
+public class Homework20 extends BaseTest {
+        @Test
+        public void deletePlaylist()   {
+            String deletedPlaylistMsg = "Deleted playlist";
+            login("jose@test.com", "te$t$tudent");
+            openPlaylist();
+            clickDeletePlaylistBtn();
+            Assert.assertTrue(getDeletedPlaylistMsg().contains(deletedPlaylistMsg));
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/229568360-8ce947b2-7bc8-4065-9d0d-ba9dae3b1980.png)

HINT

![image](https://user-images.githubusercontent.com/104761482/229568441-a471e695-ed6b-4f28-9e87-64df3f598743.png)

Please, use `assertEquals` will give you more information for example if you compare expectedNamePlayList and actualNamePlayList, it will show exact match. But assertTrue will only show the result, like expected true, found false